src/init.cpp

```diff
@@ -1317,6 +1317,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
         }
         for (int n = 0; n < NET_MAX; n++) {
             enum Network net = (enum Network)n;
```

Comment:
please convert this c-style cast to a static_cast, since you are touching the line below
```suggestion
            enum Network net = static_cast<enum Network>(n);
```

Reply:
Thanks, but this is unrelated to the changes here and I prefer to not touch it.